RELEASE.md

```diff
@@ -0,0 +1,256 @@
+# Release Documentation
+
+This document describes the process on how to release a new version of Valkey-GLIDE, how to use the GLIDE's CD workflows, and how to release a new wrapper.
+
+## GLIDE - How to release a new version
+
+**Contact a Valkey-GLIDE maintainer** on how to trigger the CD workflows and on how to login to the package manager accounts and how to access the relevant secrets.
+
+## Deployment Guide
+
+1. CHANGELOG: go through the CHANGELOG.md file found in the root folder and verify all required changes / bug fixes / features for this release are found in the file. Add a headline of the version and the release date to close all of its changes, for example for releasing version 0.3.0:
+
+    ### 0.3.0 (2024-03-25)
+    #### Changes
+    * Python: Allow routing Cluster requests by address. ([#1021](https://github.com/aws/glide-for-redis/pull/1021))
+    * Python, Node: Added HSETNX command. ([#954](https://github.com/aws/glide-for-redis/pull/954), [#1091](https://github.com/aws/glide-for-redis/pull/1091))
+
+    #### Features
+    * Python: Allow chaining function calls on transaction. ([#987](https://github.com/aws/glide-for-redis/pull/987))
+
+    ### 0.2.0 (2024-02-11)
+    #### Changes
+    * Python, Node: Added ZCARD command ([#871](https://github.com/aws/glide-for-redis/pull/871), [#885](https://github.com/aws/glide-for-redis/pull/885))
+
+    b. Open a PR to the main branch with the missing changes and the new version header, and continue after it’s merged.
+
+
+2. **If it’s a new minor/major version**, create a new branch named in the format `v$MAJOR_VERSION.$MINOR_VERSION` (e.g., v0.2), and checkout only the relevant code for this release (probably the latest commit in main, including the CHANGELOG latest commit). <br>
+
+    MAJOR_VERSION = 0 <br>
+    MINOR_VERSION = 3 <br>
+    git checkout -b `v$MAJOR_VERSION.$MINOR_VERSION` <br>
```

Comment:
Probably should all be a code block. It looks strange as it is:
<img width="906" alt="image" src="https://github.com/user-attachments/assets/0865d4ee-e70c-4ed7-93ce-a036ce64234b">